### DISCUSSION TITLE
Unable to use iTerm2 due to denied permission

### DISCUSSION THREAD

**Discussion by cyberduck** [iterate-ch/cyberduck#14418] [Q&A]
> **155c240** created the issue

I am not able to open iTerm2 terminal due to Disk permissions issue although I provided full disk access to Cyberduck. 

Could you help on how to solve the issue?

[[Image()]]

I bought Cyberduck from App Store. Could you please provide me a registration number to be able to install it from website and check back again ? 
___
Attachments

* [`Capture d’écran 2021-06-21 à 09.02.11.png`](https://github-cyberduck-assets.s3.amazonaws.com/11712/Capture%20d%E2%80%99%C3%A9cran%202021-06-21%20%C3%A0%2009.02.11.png) (21.4 KiB)

**Comment by tatze96** [comment]
It is expected that you can't use the _Open in Terminal_ option while using the App Store version of Cyberduck as this functionality isn't available within the sandboxed environment.

Regarding the license:
We don't address licensing questions within the public issue tracker and discussions page. Please send us an email to support@cyberduck.io concerning the license question if you are still interested in a registration key for the standalone version of Cyberduck.

**Comment by tatze96** [comment]
Duplicates #7664
